src/platforms/unity/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Unity
```

Comment:
Can you not adopt the common content here? https://docs.sentry.io/contributing/approach/write-getting-started/

Reply:
I'll try. This SDK has nothing common with any other but I assume a "install, configure, test" I anyway added here so I can split it in 3 files.